src/app/util/util.c

```diff
@@ -1400,3 +1400,19 @@ uint8_t emberAfMake8bitEncodedChanPg(uint8_t page, uint8_t channel)
         return channel | ENCODED_8BIT_CHANPG_PAGE_MASK_PAGE_0;
     }
 }
+
+EmberStatus emberAfScheduleServerTick(uint8_t endpoint, EmberAfClusterId clusterId, uint32_t delayMs)
+{
+    return EMBER_SUCCESS; // Stub for now.
```

Comment:
@rwalker-apple : do you have an issue tracking the work to moving silabs onto chip timers

Reply:
project-chip#658 is the closest